gems/sorbet-runtime/lib/types/private/class_utils.rb

```diff
@@ -73,24 +73,28 @@ def to_s
   # overriding it (if it is defined by one of mod's ancestors). Returns a ReplacedMethod instance
   # on which you can call `bind(...).call(...)` to call the original method, or `restore` to
   # restore the original method (by overwriting or removing the override).
-  def self.replace_method(mod, name, &blk)
+  def self.replace_method(mod, name, specific: false, &blk)
```

Comment:
There are ~5 places where we call this method. Is there a valid use case for keeping this as a parameter vs making the code either always or never some codepath?